portal-api/src/utils/string.js

```diff
@@ -7,12 +7,15 @@ const isEmptyString = (str) => {
   return false;
 };
 
-const hasValue = (str) => {
-  if (str && !isEmptyString(str)) {
-    return true;
-  }
-  return false;
-};
+/**
+ * Check if a param has a value, i.e
+ * - the param is defined
+ * - the param is not an empty string
+ * - the param is not a 'Select value' string
+ * @param {unknown}
+ * @returns Boolean
+ */
+const hasValue = (str) => str && !isEmptyString(str) && str !== 'Select value';
```

Comment:
Need to revert this one liner change @AlexBramhill - with the one liner, it can return a string instead of a boolean, this then breaks other things.

Reply:
Oh -- that's very fair. Could also do it with `!!str && !isEmptyString(str) && str !== 'Select value'`, but easy with revert